<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDialog</class>
 <widget class="QDialog" name="SerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="OkCancel_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="Second_Line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>450</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add a radio module</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Status_Browser">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>31</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select and configure the Serial/USB port where the radio module is connected to.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pixmap2_Label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="src.qrc">:/image/error.png</pixmap>
   </property>
  </widget>
  <widget class="QRadioButton" name="SelectPort_Radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ProvidePort_Radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QListView" name="Port_ListView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Port_LineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshPort_Button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="BaudRate_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="DataBits_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits</string>
   </property>
  </widget>
  <widget class="QLabel" name="Parity_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QLabel" name="StopBits_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits</string>
   </property>
  </widget>
  <widget class="QLabel" name="FlowCtrl_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>417</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Flow Control</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BaudRate_Box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DataBits_Box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Parity_Box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="StopBits_Box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FlowCtrl_Box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="Program_CheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetDefault_Button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pixmap_Label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="src.qrc">:/image/header_add_device.png</pixmap>
   </property>
  </widget>
  <widget class="Line" name="First_Line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>420</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>OkCancel_Button</zorder>
  <zorder>Second_Line</zorder>
  <zorder>First_Line</zorder>
  <zorder>Text_label</zorder>
  <zorder>Pixmap_Label</zorder>
  <zorder>Status_Browser</zorder>
  <zorder>Pixmap2_Label</zorder>
  <zorder>Pixmap_Label</zorder>
  <zorder>First_Line</zorder>
  <zorder>SelectPort_Radio</zorder>
  <zorder>ProvidePort_Radio</zorder>
  <zorder>Port_ListView</zorder>
  <zorder>Port_LineEdit</zorder>
  <zorder>RefreshPort_Button</zorder>
  <zorder>BaudRate_Label</zorder>
  <zorder>DataBits_Label</zorder>
  <zorder>Parity_Label</zorder>
  <zorder>StopBits_Label</zorder>
  <zorder>FlowCtrl_Label</zorder>
  <zorder>BaudRate_Box</zorder>
  <zorder>DataBits_Box</zorder>
  <zorder>Parity_Box</zorder>
  <zorder>StopBits_Box</zorder>
  <zorder>FlowCtrl_Box</zorder>
  <zorder>Program_CheckBox</zorder>
  <zorder>SetDefault_Button</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OkCancel_Button</sender>
   <signal>accepted()</signal>
   <receiver>SerialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkCancel_Button</sender>
   <signal>rejected()</signal>
   <receiver>SerialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
